stable/opa/Chart.yaml

```
@@ -6,7 +6,7 @@ keywords:
 - opa
 - admission control
 - policy
-version: 0.2.0
+version: 0.3.0
```

since this chart is in the stable folder, lets make it 1.0.0 please
```suggestion
version: 1.0.0
```